physionet-django/user/models.py

```diff
@@ -444,6 +444,15 @@ def has_orcid(self):
             pass
         return False
 
+    def get_orcid(self):
```

Comment:
perhaps rename to `get_orcid_id` and return the ID instead of the object?

Reply:
Ah yes, my bad. at first, I had directly accessed the orcid_id and later I opted to use the get method and forgot to get the actual id 🤦 Sorry about the issue.

Will push the change first thing in the morning.